src/main/java/com/zendesk/maxwell/bootstrap/SynchronousBootstrapper.java

```diff
@@ -182,8 +189,8 @@ private ResultSet getAllRows(String databaseName, String tableName, Schema schem
 	}
 
 	private Statement createBatchStatement(Connection connection) throws SQLException, InterruptedException {
-		Statement statement = connection.createStatement();
-		statement.setFetchSize(context.getConfig().bootstrapperBatchFetchSize);
+		Statement statement = connection.createStatement(ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY);
+		statement.setFetchSize(Integer.MIN_VALUE);
```

Comment:
sounds good, as an FYI before making the streaming change I also tried bumping up to the latest connector version (5.1.38) with no luck. That version itself as a drop in seemed to work fine but didn't fix the cursor fetching.

I also tried the following once the jdbc_options patch was in place with no luck

netTimeoutForStreamingResults=3600&sessionVariables=wait_timeout=3600&useUnbufferedInput=true&useReadAheadInput=false